code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -262,14 +263,15 @@
 // RIOT SHOTGUN //
 /obj/item/gun/ballistic/shotgun/riot //for spawn in the armory
 	name = "light shotgun"
-	desc = "A light weight shotgun chambered in 12 gauge, designed to be used by paramilitary companies during air raids, but was later absorbed by Vault-tec for use by security in their vaults."
+	desc = "A lightweight shotgun chambered in 12 gauge, designed to be used by paramilitary companies during air raids, but was later absorbed by Vault-tec for use by security in their vaults."
 	icon_state = "riotshotgun"
 	item_state = "shotgunriot"
-	force = 40
+	force = 25
```

Comment:
stealth nerf to smashing people in the face with heavy weapons?

Reply:
"Light" shotgun
40 force damage

I don't think it should be a robust melee weapon on top of the highest capacity shotgun that's commonly found.

Reply:
yeah originally it had a desc of like "this is a shotgun with a super bulky reinforced stock of cracking heads" but someone changed it

Reply:
Designed to be a buff melee weapon, and was autistically renamed by oxford, please change back, keeping in mind most armors will bring it down to about 20/25